docs/topics/asyncio.rst

```diff
@@ -98,6 +98,52 @@ Futures. Scrapy provides two helpers for this:
          into your own code.
 
 
+Async additional requests
+=========================
```

Comment:
:lipstick: No strong opinion, but I wonder if we could find a better title, since technically yielding a request from a callback also gives you an async additional request. Maybe “Inline requests”?

Reply:
Added suggestion.